sleap/io/visuals.py

```diff
@@ -515,6 +533,7 @@ def save_labeled_video(
         fps: Frames per second for output video.
         scale: scale of image (so we can scale point locations to match)
         crop_size_xy: size of crop around instances, or None for full images
+        background: output video background. Either Original, Black, White, Grey
```

Comment:
Call `save_labeled_video` with background argument in [`sleap.io.visuals.main`](https://github.com/talmolab/sleap/blob/045ff9d113ac0434e434b3a9c548d55ddb70188d/sleap/io/visuals.py#L650)